tests/driver.c

```diff
@@ -217,6 +251,34 @@ void test_receive_mam_message(void) {
   printf("Average time of receive_mam_message: %lf\n", sum / TEST_COUNT);
 }
 
+void test_iri_proxy_apis() {
+  for (int i = 0; i < iri_proxy_apis_num; i++) {
+    char* json_result;
+    double sum = 0;
+
+    if (!strcmp(iri_prxoy_apis_g[i].name, "attach_to_tangle") || !strcmp(iri_prxoy_apis_g[i].name, "get_neighbors")) {
```

Comment:
Shall we allow the two calls as IRI proxy?

Reply:
I think we could provide these two API, but whether do they have effect on a certain IRI, could be decide by the maintainer of each IRI.

Reply:
Before nested IRI proxy calls, we need to make a comprehensive list on API and use scenario.

Reply:
After discussion, use scenario of `attach_to_tangle` and `get_neighbors` are not explicit, will it be better to remove these IRI Proxy API in this PR?

Reply:
I defer to @HowJMay for API coverage.